src/components/GraphFilter/GraphFilter.tsx

```diff
@@ -9,15 +9,17 @@ export default class GraphFilter extends React.Component<GraphFilterProps, Graph
   // TODO:  We should keep these mappings with their corresponding filtering components.
   // GraphFilter should be minimal and used for assembling those filtering components.
   static INTERVAL_DURATION = [
```

Comment:
I might arrive late into this PR. I am wondering if both duration arrays can be unified into one. If kiali is going to share state of duration, would it make sense to share same DURATION array?

Reply:
Yes, but I don't how I shoudl put this array, I don't know If have sense to store it in config.ts